module-3/back-end/src/models/project.js

```diff
@@ -0,0 +1,47 @@
+const {knex} = require('../db/knex')
+const {canFindById, canGetAll, canDeleteById} = require('./baseModel')
+
+const config = {
+    tableName: 'project'
+}
+
+const Model = (config) => {
+
+    const create = async project => {
+
+        return await knex(config.tableName).insert(project, [
+            'id',
+            'title',
+            'description',
+            'document_id',
```

Comment:
'document_id' what is it for? If it's supposed to be a relationship between project and documents, I'd say it should read as plural `documents_id`, but either way I think it's better to have a separate table to store relationships between projects and documents, like `project_documents` for example and the structure of that table would be as follows
 - `id`
 - `document_id`
 - `project_id`

Reply:
I've changed structure